pyispyb/core/modules/events.py

```diff
@@ -47,6 +47,8 @@ class EntityType:
                 models.DataCollection.DataCollectionGroup,
                 models.DataCollectionGroup.Workflow,
             ],
+            models.DataCollection.SSXDataCollection,
```

Comment:
It'd be sensible to make this optional based on if models.DataCollection.SSXDataCollection exists. That way if people dont have the table it doesnt break the app